wolfcrypt/src/tfm.c

```diff
@@ -637,11 +632,8 @@ int fp_mul_comba(fp_int *A, fp_int *B, fp_int *C)
 
    IF_HAVE_INTEL_MULX(ret = fp_mul_comba_mulx(A, B, C), return ret) ;
 
-#ifdef WOLFSSL_SMALL_STACK
-   tmp = (fp_int*)XMALLOC(sizeof(fp_int), NULL, DYNAMIC_TYPE_BIGINT);
-   if (tmp == NULL)
-       return FP_MEM;
-#endif
+   WC_ALLOC_VAR_EX(tmp, fp_int, 1, NULL, DYNAMIC_TYPE_BIGINT,
```

Comment:
Same.. should `WC_DECLARE_VAR` be used above?

Reply:
fixed